charts/artifact-caching-proxy/templates/statefulset.yaml

```diff
@@ -27,9 +32,13 @@ spec:
               ]
             }
           }
-        {{- if .Values.logCollection.enabled }}
-        # log collection with Datadog
-        ad.datadoghq.com/{{ .Chart.Name }}.logs: {{ .Values.logCollection.config | squote }}
+        {{- end }}
+        {{- if .Values.datadog.logCollection.enabled }}
+        # Nginx Log collection with Datadog:
+        # - https://docs.datadoghq.com/integrations/nginx/?tab=kubernetes#log-collection
+        # - https://docs.datadoghq.com/containers/kubernetes/log/?tab=kubernetes#configuration
+        #   1. With 'ad.datadoghq.com/xxx.logs' , the 'xxx' must be the name of the pod's container name
+        ad.datadoghq.com/{{ .Chart.Name }}.logs: '[{"source":"nginx","service":"{{ .Chart.Name }}","tag": "env:prod"}]'
```

Comment:
This change bothers me a bit as I think it should be configurable depending on where you deploy it. Ex: the tags.

Reply:
I propose to open a subsequent PR to allow specifying the tags as a value.

But for now, the "raw" specification of the 0.4.4 version is a bug from my view: if the template changes the container names, then the chart installation are broken.